earth_enterprise/rpms/build.gradle

```diff
@@ -22,6 +25,16 @@
 // This Scons target copies build artifacts, binaries and documents into the
 // /tmp/fusion_os_install path.
 //
+//     ./gradlew -Prelease
+// Build the RPMs using the files in 'NATIVE-REL-x86_64'
```

Comment:
I don't see any code to process a `release` property.  I think this comment about `-Prelease` should be removed.

Reply:
@tst-dmendyke: This minor comment still stands.  There are only `optimize` and `debug` project properties now.